recipes/ditasic/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: ditasic
+  version: "0.2"
+
+source:
+  url: https://gitlab.com/rki_bioinformatics/ditasic/uploads/9c6dd7f3bfde8d958a03c49ba89276d9/ditasic_0.2.tar.gz
+  md5: 27fbe916e14bf2c47fc215b988dcfb27
+
+build:
+  number: 0
+  skip: True # [not py3k]
```

Comment:
`skip: True` can be removed if the  `host` and `run` sections specify `python >=3`. This change will probably cause the linter to suggest adding `noarch: python` to the `build` section.